core/src/main.cpp

```diff
@@ -946,11 +958,17 @@ auto main(int _argc, char* _argv[]) -> int
 		}
 
 		if (vm.count("config-file") == 0) {
-			fmt::print(stderr, "Error: Missing [CONFIG_FILE_PATH] parameter.");
+			fmt::print(stderr, "Error: Missing [CONFIG_FILE_PATH] parameter.\n");
```

Comment:
Do we have `fmt::println(...)`?

Reply:
I don't think the version of fmtlib we use supports that version of print.

_I wish it did, but oh well._